### PR TITLE
Added __codename__

### PR DESCRIPTION
**Pull request by micheles**
Now it will appear in the calculation logs, see https://ci.openquake.org/job/zdevel_oq-engine/1947

### CODE REVIEW

File: openquake/risklib/__init__.py [hunk 1]
```diff
@@ -33,3 +33,5 @@
     sys.exit('Found an obsolete version of commonlib; '
              'please remove %s and/or fix your PYTHONPATH'
              % os.path.dirname(path))
+
+__codename__ = 'Keiiti Aki'
```
Comment (daniviga) [inline openquake/risklib/__init__.py]: We should keep the `__codename__` empty and populate it only during the release process.

Comment (micheles) [inline openquake/risklib/__init__.py]: Agreed

**Comment by daniviga** [comment]
LGTM

**Comment by micheles** [comment]
After discussion with Daniele we decided to not put the `__codename__` in the code for the moment, just in the README and in the communications.

**Pull request closed without merging**